api/v0/runtime.proto

```diff
@@ -68,6 +65,31 @@ service Spawn {}
 /// * Spawn      A special function that creates a child instance with inherited
 /// properties of the parent
 
+/// A pod is a higher level abstraction than Aurae cells, and to most users
+/// will look at feel like one or more "containers".
+///
+/// Pods will run an OCI compliant container image.
+///
+/// A pod is a group of one or more containers with shared network and storage.
+service PodService {
+
+  rpc Allocate(AllocatePodRequest) returns (AllocatePodResponse) {}
+  rpc Start(StartContainerRequest) returns (StartContainerResponse) {}
+  rpc Stop(StopContainerRequest) returns (StopContainerResponse) {}
```

Comment:
I still prefer these to be more verbose in their naming. E.g., `AllocatePod`, `StartContainer`, `StopContainer`, `FreePod`.

Not a big deal though